h/static/scripts/group-forms/components/EditGroupMembersForm.tsx

```diff
@@ -49,24 +72,52 @@ export default function EditGroupMembersForm({
     }
     const abort = new AbortController();
     setErrorMessage(null);
-    fetchMembers(config.api.readGroupMembers, abort.signal)
+    fetchMembers(config.api.readGroupMembers, userid, abort.signal)
       .then(setMembers)
       .catch(err => {
         setErrorMessage(`Failed to fetch group members: ${err.message}`);
       });
     return () => {
       abort.abort();
     };
-  }, [config.api.readGroupMembers]);
+  }, [config.api.readGroupMembers, userid]);
 
   const columns = [
     {
       field: 'username' as keyof MemberRow,
       label: 'Username',
     },
+    {
+      field: 'canDelete' as keyof MemberRow,
+      label: '',
+      classes: 'w-[55px]',
+    },
   ];
 
   const [filter, setFilter] = useState('');
+  const [pendingRemoval, setPendingRemoval] = useState<string | null>(null);
+
+  const removeUserFromGroup = async (username: string) => {
+    // istanbul ignore next
+    if (!members || !config.api.removeGroupMember) {
+      return;
+    }
+    const member = members.find(m => m.username === username);
+    // istanbul ignore next
+    if (!member) {
+      return;
+    }
```

Comment:
If we know for sure the member is going to be found, maybe we can simply use a non-null assertion.

```suggestion
    const member = members.find(m => m.username === username)!;
```